spec/client_spec.js

```diff
@@ -36,6 +36,85 @@ describe('Client', function() {
     window.addEventListener.callArgWith(1, evt);
   }
 
+  describe('isDomainValid', function() {
+    it('Should instantiate client when domain is valid 1', function() {
```

Comment:
Could we refer to the domain in the description, rather than a count, just so it's easier understand when they fail?